octodns_gcore/__init__.py

```diff
@@ -283,12 +282,25 @@ def _data_for_CNAME(self, _type, record):
 
     def _data_for_multiple(self, _type, record):
         extra = dict()
-        if record.get("filters") is not None:
-            pools, rules, defaults = self._data_for_dynamic(record)
-            extra = {
-                "dynamic": {"pools": pools, "rules": rules},
-                "values": defaults,
-            }
+        filters = record.get("filters")
+        if filters is not None:
+            filter_types = [filter['type'] for filter in filters]
+            if 'geodns' in filter_types:
+                pools, rules, defaults = self._data_for_dynamic_geo(record)
+                extra = {
+                    "dynamic": {"pools": pools, "rules": rules},
+                    "values": defaults,
+                }
+            else:
+                # other type should be "healthcheck"
+                pools, rules, octodns, defaults = (
+                    self._data_dynamic_healthcheck(record)
+                )
+                extra = {
+                    'dynamic': {'pools': pools, 'rules': [{'pool': 'pool-0'}]},
```

Comment:
Is `pool-0` something specific to the provider or just picked here b/c a name was needed?

Reply:
It's not specific. I picked the same name used in the records created by Gcore GUI. And the same as the already geodns healthcheck support.